### PR TITLE
Weather update

### PR DESCRIPTION
**Pull request by matcracker**
Check if all is correct without conflicts

### CODE REVIEW

**Comment by matcracker** [comment]
Ok I can merge :+1:

File: src/pocketmine/Player.php [hunk 1]
```diff
@@ -134,7 +134,6 @@
 use pocketmine\tile\Tile;
 use pocketmine\utils\TextFormat;
 use raklib\Binary;
-use pocketmine\level\weather\WeatherManager;
```
Comment (thebigsmileXD) [inline src/pocketmine/Player.php]: Why removed the WeatherManager? Isn't this necessary?

File: src/pocketmine/command/defaults/WeatherCommand.php [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        );
+        $this->setPermission("pocketmine.command.weather.clear;pocketmine.command.weather.rain;pocketmine.command.weather.thunder");
+    }
+
+    public function execute(CommandSender $sender, $currentAlias, array $args){
+        if(!$this->testPermission($sender)){
+            return true;
+        }
+
+        if(count($args) > 2 || count($args) === 0){
```
Comment (thebigsmileXD) [inline src/pocketmine/command/defaults/WeatherCommand.php]: In vanilla Minecraft theres also /weather clear 30 or /weather rain 3 to make the weather like this for that specific time

Comment (thebigsmileXD) [inline src/pocketmine/command/defaults/WeatherCommand.php]: I am sorry, just saw you also did that xD

File: src/pocketmine/Player.php (hunk 1 above)
Comment (matcracker) [inline src/pocketmine/Player.php]: No, I delete the file, I completly rewrited the weather core. One thing, can you bump API to 1.13.2? At the moment I'm from phone

File: src/pocketmine/command/defaults/WeatherCommand.php (hunk 2 above)
Comment (matcracker) [inline src/pocketmine/command/defaults/WeatherCommand.php]: :+1:

**Comment by thebigsmileXD** [comment]
Delete branch?

**Comment by matcracker** [comment]
Yes, thanks :)

**Comment by thebigsmileXD** [comment]
Okay, have a good night!

**Comment by matcracker** [comment]
Thank you, have a good night you too

**Pull request merged**